src/App.tsx

```diff
@@ -1,26 +1,102 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+/* eslint-disable prettier/prettier */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 
-const USER_ID = 0;
+import * as React from 'react';
+import { useEffect, useState, useRef } from 'react';
+import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { FilterStatus, ErrorMessages } from './types';
+import { Todo, USER_ID } from './types';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoList } from './components/TodoList';
+import { filterTodos } from './utils/fiterTodos';
+import { ErrorNotification } from './components/ErrorNotification';
 
 export const App: React.FC = () => {
+  const [isDisabledInput, setDisableInput] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isDisabledInput, setIsDisabledInput] = useState<boolean>(false);
```

set function should have same name as variable but with `set`